views/js/qtiCreator/helper/gridUnits.js

```diff
@@ -155,31 +155,31 @@ define(['lodash'], function(_){
             //too much !
 
             //@todo : start with the hightest refactored
-            _.eachRight(positive, function(col){
+            for (let i = positive.length - 1; i >= 0; i--) {
```

Comment:
In this case we cant, because we need break look in this condition `if(totalRefactoredUnits === max)`, and forEachRight do not support return false, or break.